src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+/**
+ * Represents a <code>Parser</code> which parses input strings
+ * to <code>Duke</code> and returns the appropriate enum type
+ * defining the command.
+ */
+
+
+public class Parser {
+
+    public Parser() {}
+
+    /**
+     * Parses input string to return appropriate enum type.
+     * @param input
```

Comment:
I think the description can be a bit more concrete. i.e. the string input that the user types